ocaml/gencert/selfcert.ml

```diff
@@ -93,21 +95,25 @@ let selfsign issuer extensions key_length days certfile =
           x
     with e ->
       let msg =
-        Printf.sprintf "generating RSA key for %s failed: %s" certfile
-          (Printexc.to_string e)
+        Printf.sprintf "generating RSA key failed: %s" (Printexc.to_string e)
```

Comment:
This will loose the original stacktrace.
I would suggest to add a `Backtrace.is_important e` prior to `let msg` to ensure that the stacktrace is saved and we see the full stacktrace in the logs, even if we change the error message here